### ISSUE TITLE
Overfitting in the plots

### ISSUE THREAD

**Issue by sfarkya** [vietnguyen91/Yolo-v2-pytorch#12]
Hi @vietnguyen91 , thanks for the great repo. 

I was looking at the plots you have shown for the VOC training and I see a lot of overfitting in the test data. So, I was wondering why is that? Are you able to get good mAP eventually on the test data. 

I am writing the code for yolo v2 myself in tensorflow and that's an issue I am really suffering from that my model is consistently overfitting just like yours and my output aren't as good as you have shown. I am using adam optimizer. I tried using SGD with momentum and weight decay as well as adam with weight decay but my network doesn't learn anything and the training oscillates without convergence. Have you seen anything of this sort? 

Thanks for any suggestion or comments. 
Best, 
Saurabh

**Comment by TimofeyKruk** [comment]
I have kind of the same problem! The mAP is about zero and only people are detected quite well. If you found the reason why it happens I am looking forward to hearing it! Thank you in advance!
